docs/intro/tutorial.rst

```diff
@@ -265,7 +265,7 @@ element, including its tags:
 >>> response.css('title').getall()
 ['<title>Quotes to Scrape</title>']
 
-The other thing is that the result of calling ``.getall()`` is a list: it is
+The second is that the result of calling ``.getall()`` is a list: it is
```

Comment:
Since there is not “the first” in earlier text, I think the proposed change is wrong, and the existing text is correct.